docs: Add common use cases (#4677)
CloudFire-LLC/cloudfire: website/src/app/kb/use-cases/private-network-access/readme.mdx

import SupportOptions from "@/components/SupportOptions";
import Alert from "@/components/DocsAlert";
import Image from "next/image";
import Link from "next/link";
import PlanBadge from "@/components/PlanBadge";

<PlanBadge plans={["starter", "team", "enterprise"]}>

# Access a Private Network

</PlanBadge>

In this guide, we'll be using Firezone to secure access to a private subnet
behind a firewall.

This is useful when you have hosts or services behind a firewall that you want
to keep secure, but still need to access it from external networks like the
internet.

<Alert color="warning">
  In general, we recommend using a more granular approach to secure access using
  either [DNS](/kb/use-cases/web-app-access) or
  [IP-based](/kb/use-cases/host-access) Resources instead of the blanket

>   approach used in this guide. We recommend using this guide only if using DNS
>   or IP-based Resources is not feasible, or if you want a stepping stone towards
>   a more granular approach that still works with existing systems today.

Reviewer: ```suggestion
  approach used in this guide. Only use this guide if using DNS
  or IP-based Resources is not feasible, or if you need a stepping stone towards a more granular approach.
```
nit

</Alert>

## Prerequisites

> - A Site that contains the subnet you want to secure access to.

Reviewer: Same as above, "Site that _will_ contain" or similar.
